### PR TITLE
Feature/206 test committee accounts for filing

### PR DESCRIPTION
**Pull request by dheitzer**
see api pr description.

### CODE REVIEW

File: front-end/src/app/shared/services/fec-api.service.ts [hunk 1]
```diff
@@ -36,14 +39,10 @@ export class FecApiService {
     if (!committeeId) {
       throw new Error('No Committee Id provided.');
     }
-    const headers = this.getHeaders();
```
Comment (toddlees) [inline front-end/src/app/shared/services/fec-api.service.ts:60]: Weren't we planning on making this configurable.  We don't want to eliminate the actual code that works, but point to something else when it's configured to mock right?

File: front-end/src/app/shared/services/fec-api.service.ts [hunk 2]
```diff
@@ -60,48 +59,9 @@ export class FecApiService {
...
```
Comment (toddlees) [inline front-end/src/app/shared/services/fec-api.service.ts:84]: We still need this code right?  we just need to configure the endpoints it hits to be mocks when the system is configured to test

**Review by toddlees — DISMISSED** [review]

**Comment by dheitzer** [comment]
Made the changes discussed in our dev meeting for api:
1) using an environment variable to determine committee id data to override
2) using a local json file to override the committee data for specified committee ids

File: front-end/src/app/shared/services/fec-api.service.ts (hunk 2 above)
Comment (dheitzer) [reply front-end/src/app/shared/services/fec-api.service.ts:84]: made changes per our dev meeting for the new approach to overriding committee data in the api.  I believe this is OBE now.

File: front-end/src/app/shared/services/fec-api.service.ts (hunk 1 above)
Comment (dheitzer) [reply front-end/src/app/shared/services/fec-api.service.ts:60]: made changes per our dev meeting for the new approach to overriding committee data in the api.  I believe this is OBE now.

**Review by mjtravers — APPROVED** [review]

**Pull request merged**